pcsx2/gui/MainFrame.cpp

```diff
@@ -584,19 +587,66 @@ void MainEmuFrame::OnActivate( wxActivateEvent& evt )
 }
 // ----------------------------------------------------------------------------
 
+static void GetMenuItemLabel(MenuIdentifiers Id, wxString* label)
```

Comment:
> but you have then navigate to SetMenuItemLabel to determine what label is being used:

I think you mean ``GetMenuItemLabel()``? It's literally just one line before ``MainEmuFrame::ApplyCoreStatus()`` and I don't see how navigating to a function to view it's content is a hard thing. I would prefer to separate these label string initialization to a separate function to reduce the mess in  ``MainEmuFrame::ApplyCoreStatus()`` as stated earlier.